assets/dev/js/editor/components/template-library/event-manager/index.js

```diff
@@ -17,9 +17,34 @@ const EVENTS_MAP = {
 	PAGE_VIEWED: 'page_viewed',
 	DELETION_UNDO: 'deletion_undo',
 	CT_RECORDING_START: 'ctemplates_session_replay_start',
+	CT_BADGE_HOVER: 'ct_badge_hover',
+};
+
+const CLOUD_TEMPLATES_EXPERIMENTS = {
+	SAVE_TEMPLATE: 'save-template-cloud',
 };
 
 export class EventManager {
+	getExperimentVariant( experimentName ) {
+		if ( ! elementorCommon?.eventsManager ) {
+			return 'control';
+		}
+
+		return elementorCommon?.eventsManager?.getExperimentVariant( experimentName, 'control' ) || 'control';
```

Comment:
Do we want to control this from Editor Assets as well? What if we want to stop A/B testing at any point in time?

@ManorHazaz @max-zu